pkg/resources/tag.go

```diff
@@ -287,9 +308,48 @@ func UpdateTag(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	// If there is change in allowed_values field
+	if d.HasChange("allowed_values") {
+		if _, ok := d.GetOk("allowed_values"); ok {
+			_, n := d.GetChange("allowed_values")
```

Comment:
_ , v is more common than using _, n, especially if there is no apparent reason to use "n"